estuary/utils/general.py

```diff
@@ -462,5 +462,5 @@ def _get_plural_label(label):
         result = '{0} {1} {2}'.format(rel_label, relationship, story_node_display_name)
         return result[0].upper() + result[1:]
     else:
-        raise RuntimeError('A node with the label {0} does not have a {1} relationship'.format(
-            story_node_story_flow['display_label'], rel_direction))
+        raise RuntimeError('A node of this type does not have a {0} relationship'.format(
```

Comment:
hm, can we use `story_node_display_name` instead?

Reply:
I was thinking that but I thought it'd add to the confusion.

Reply:
I just feel not having the specific name might just add to confusion.